specification/postgresql/resource-manager/readme.typescript.md

```diff
@@ -7,7 +7,26 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/arm-postgresql"
```

Comment:
There are two outputs:
```
$(typescript-sdks-folder)/sdk/postgresql/arm-postgresql-flexibleserver/
$(typescript-sdks-folder)/sdk/postgresql/arm-postgresql/
```
But they use the same package name, which is unreasonable. They should use different package name.

Reply:
```suggestion
```